### PR TITLE
test: Test for disable_new_netns=false

### LINKED ISSUE #978: Create a test for disable_new_netns=false

### PR DESCRIPTION
**Pull request by GabyCT**
Create a test to run without network and configure disable_new_netns
to false at the configuration.toml.

Fixes #978

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

**Comment by jodh-intel** [comment]
lgtm

File: integration/network/disable_net/net_none.bats [hunk 1]
```diff
@@ -16,11 +20,33 @@ setup () {
...
-	IMAGE="busybox"
-	PAYLOAD="tail -f /dev/null"
-	NAME="test"
+@test "Disable_new_netns equal to false" {
+	extract_kata_env
+
+	sudo sed -i 's/#disable_new_netns = true/disable_new_netns = false/g' ${RUNTIME_CONFIG_PATH}
+
+	# Run a container with no network
+        docker run -d --runtime=${RUNTIME} --name=${NAME} --net=none ${IMAGE} ${PAYLOAD}
```
Comment (devimc) [inline integration/network/disable_net/net_none.bats]: please, use tabs

Comment (GabyCT) [reply integration/network/disable_net/net_none.bats]: @devimc done :)

**Pull request merged**